contrib/resources/translations/README.md

```diff
@@ -13,25 +13,27 @@ dosbox -lang en -c "config -wl en.lng" -c "exit"
 or do the following:
 
 1. In the configuration file, go to the `[dosbox]` section and set`language = en`.
-2. Run the most current version of DOSBox Staging.
-3. Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
+1. Run the most current version of DOSBox Staging.
+1. Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
   path, instead of simply `en.lng`).
 
 This will produce a file with the current strings to translate, use it as a
 reference.
 
 ## Updating the translation
 
-Update a given translation by editing the corresponding `*.lng` file with a text editor
-and saving it with UTF-8 encoding and Unix line endings.
+Update a given translation by editing the corresponding `*.lng` file with a text
+editor and saving it with UTF-8 encoding and Unix line endings.
 
 Suitable editors are often called ASCII editors, code editors, etc. - Microsoft Word
-is not suitable, but Notepad (Windows), Notepad++ (Windows), or Kate (Linux) will do.
+is not suitable, but Notepad (Windows), Notepad++ (Windows), or Kate (Linux)
+will do.
```

Comment:
Suggest mentioning TextMate for macOS (free & open-source and quite good).

Reply:
OK, mentioned.